Resources/Prototypes/Entities/Objects/Weapons/Melee/pickaxe.yml

```diff
@@ -39,7 +39,7 @@
   - type: GatheringTool
     damage:
       types:
-        Piercing: 25
+        Structural: 75
```

Comment:
doesn't this make it practically a fireaxe.

Reply:
This isn't the damage like a weapon. This is the damage that's applied to gatherables. This doesn't apply to walls, only to asteroids and other gatherables. The actual damage of the weapon is still just 14.